### ISSUE TITLE
Permission error occured when accessing the comments of a specific issue

### ISSUE THREAD

**Issue by k1won** [openvinotoolkit/cvat#4416]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour

- returns all comments of a specific issue
### Current Behaviour

- permission error occured (except admin account)
```html
HTTP 403 Forbidden
Allow: GET, OPTIONS
Content-Type: application/vnd.cvat+json
Vary: Accept

{
    "detail": "You do not have permission to perform this action."
}
```
### Possible Solution

 - add additional 'key:value'('comments': 'view') to the get_scopes() method in IssuePermission class like below. (apps.iam.permissions.py)
```python
    @staticmethod
    def get_scopes(request, view, obj):
        return [{
            'list': 'list',
            'create': 'create@job',
            'destroy': 'delete',
            'partial_update': 'update',
            'retrieve': 'view',
            'comments': 'view'
        }.get(view.action, None)]
```
 
### Steps to Reproduce (for bugs)

1. create an issue and comment by task owner
1. access the /api/issues/{id}/comments api endpoint

### Your Environment

- Git hash commit (`git log -1`): cc98ff05
- Docker version `docker version` (e.g. Docker 17.0.05):
- Are you using Docker Swarm or Kubernetes?
- Operating System and version (e.g. Linux, Windows, MacOS): ubuntu 18.04
- Code example or link to GitHub repo or gist to reproduce problem:
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `OPA` container
```json
    "input": {
        "auth": {
            "organization": {
                "id": 9,
                "owner": {
                    "id": 6
                },
                "user": {
                    "role": "owner"
                }
            },
            "user": {
                "id": 6,
                "privilege": "business"
            }
        },
        "resource": {
            "assignee": {
                "id": null
            },
            "id": 12,
            "job": {
                "assignee": {
                    "id": 4
                }
            },
            "organization": {
                "id": 9
            },
            "owner": {
                "id": 4
            },
            "project": null,
            "task": {
                "assignee": {
                    "id": 7
                },
                "owner": {
                    "id": 6
                }
            }
        },
        "scope": null
    },
```
</summary>
    </details>

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

**Comment by nmanovic** [comment]
@k1won , thanks for the bug report! Please let me know if you would like to send us a PR. In general if I don't have a reploy from you in several hours, I will fix it by myself. I want to have the fix in our next release v2.0.0

**Comment by k1won** [comment]
I will send a PR.
Thanks.

2022년 3월 4일 (금) 오후 6:22, Nikita Manovich ***@***.***>님이 작성:
